src/app/pages/sharing/webdav/webdav-form/webdav-form.component.ts

```diff
@@ -115,25 +140,30 @@ export class WebdavFormComponent implements FormConfiguration {
         message: shared.dialog_message,
         hideCheckBox: true,
         buttonMsg: shared.dialog_button,
-      }).pipe(filter(Boolean), untilDestroyed(this)).subscribe(() => {
-        this.loader.open();
-        this.ws.call('service.update', [service.id, { enable: true }]).pipe(untilDestroyed(this)).subscribe(() => {
-          this.ws.call('service.start', [service.service]).pipe(untilDestroyed(this)).subscribe(() => {
-            this.loader.close();
-            this.dialog.info(
-              this.translate.instant('{service} Service', { service: 'WebDAV' }),
-              this.translate.instant('The {service} service has been enabled.', { service: 'WebDAV' }),
-              '250px',
-              'info',
-            ).pipe(untilDestroyed(this)).subscribe(() => {});
+      }).pipe(untilDestroyed(this)).subscribe((res) => {
+        if (!res) {
+          this.slideInService.close();
+        } else {
+          this.loader.open();
+          this.ws.call('service.update', [service.id, { enable: true }]).pipe(untilDestroyed(this)).subscribe(() => {
```

Comment:
See if it would be possible to make this flatter, i.e. switching to a different observable instead of subscribing in a subscription. May be easier to read and may have less duplicate code for error processing.